aws/src/integration/java/org/apache/iceberg/aws/s3/TestS3FileIO.java

```diff
@@ -397,6 +405,9 @@ public void testReadMissingLocation() {
 
   @Test
   public void testMissingTableMetadata() {
+    skipIfAnalyticsAcceleratorEnabled(
+        new S3FileIOProperties(properties),
+        "Analytics Accelerator Library does not support custom Iceberg exception: NotFoundException");
```

Comment:
we probably can solve this easily in the StreamWrapper right? Or would we have to change AAL library to solve this problem.

Reply:
Yes, this should be a change in StreamWrapper class rather than AAL side